app/controllers/appeal-application/home-office-details.ts

```diff
@@ -88,11 +89,8 @@ function postDateLetterSent(updateAppealService: UpdateAppealService) {
         month,
         year
       };
-      if (diffInDays <= 14) {
-        req.session.appeal.application.isAppealLate = false;
-      } else {
-        req.session.appeal.application.isAppealLate = true;
-      }
+
+      req.session.appeal.application.isAppealLate = diffInDays > 14;
```

Comment:
Don't think we wanted to change this in the end what is currently on master is correct I believe. We had a bug around BST that got fixed.